releaseNotes/PythonFeatureManagement.md

```diff
@@ -2,6 +2,72 @@
 
 [Source code][source_code] | [Samples][samples]
 
+## 2.0.0b1 - September 10, 2024
+
+### Enhancement
+
+* Added support for variant feature flags defined using [Microsoft Feature Management schema](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureManagement.v2.0.0.schema.json). Variants and telemetry can be declared using [Microsoft Feature Flag schema v2](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureFlag.v2.0.0.schema.json). The Microsoft Feature Management schema is designed to be language agnostic, which enables you to apply a consistent feature management configuration across Microsoft feature management libraries of different programming languages.
+
+### Variants
+
+Variants are a tool that can be used to surface different variations of a feature to different segments of an audience. Previously, this library only worked with flags. The flags were limited to boolean values, as they are either enabled or disabled. Variants have dynamic values. They can be string, int, a complex object, or a reference to a ConfigurationSection.
```

Comment:
Is this somewhat copied from the .NET release notes? Ross and I have worked in multiple iterations to simplify it (https://github.com/Azure/AppConfiguration/pull/974). Still WIP. I hope we don't have to do it again here :)

BTW, is there a python feature reference doc like what .NET has. If not, we should create one. The release notes are not documentation for features. We should point users to the documentaton for details.

Reply:
This was based on what is in the DotNet release notes. I figured we wanted the same thing here.

Reply:
As for the second question we have this https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-python-reference, as of a few days ago.

Reply:
The last half of the doc or so is variants and telemetry https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-python-reference?pivots=preview-version#variants.

Reply:
Very cool. Then we should point users to the doc instead of making the release note a doc.

Reply:
Updated. Though the link will break when we remove it from preview, we'll have to remember to update it when we GA.

Reply:
You can actually leave out the preview pivot and it'll work. Try adjusting to `https://learn.microsoft.com/en-us/azure/azure-app-configuration/feature-management-python-reference#variants`

Reply:
@rossgrambo thanks, fixed didn't know that.